app/serializers/data_explorer/query_serializer.rb

```
@@ -0,0 +1,16 @@
+class DataExplorer::QuerySerializer < ActiveModel::Serializer
+  attributes :id, :sql, :name, :description, :param_info, :created_at, :username, :group_ids, :last_run_at, :hidden, :user_id
+
+  def param_info
+    object.params.map(&:to_hash) rescue nil
```

Not a fan of using a blanket rescue nil because it swallows all errors.